customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDto.java

```diff
@@ -25,6 +26,7 @@
     private String institutionDns;
     private String feideOrganizationId;
     private String cristinId;
+    private Set<VocabularySetting> vocabularySettings;
```

Comment:
Here we are possibly entering into a mixture of DTO and DAO models. I do not see necessarily a danger at this point as probably the changes will not be many, but any change in the DTO will automatically mean that we should migrate our database.

Reply:
One solution would be to create  VocabularySettingDao and `VocabularySettingDto` that are extending the VocabularySetting. Later these classes could diverge if necessary. But maybe we can leave it as it is and address the problem if and when it occurs.